source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
 E.2 Audit Tests
 ---------------
 
-There are many audit tests required by the COUNTER R5 Code of Practise, detailed below.
+E.2.1 Validation Tool
+"""""""""""""""""""""
 
+Once reports are available for testing (i.e. up to 28 days after the end of the seeding month), auditors MUST run the COUNTER Reports and Standard Views of COUNTER Reports that are within the scope of the audit through the Validation Tool.
```

Comment:
This test must include all attributes content providers are required to support, including attributes that are just included in the COUNTER Reports if called for. I think this has to be stated explicitly.